src/components/Dropdowns/ColorRampDropdown.tsx

```diff
@@ -111,131 +208,64 @@ const ColorRampSelector: React.FC<ColorRampSelectorProps> = (propsInput): ReactE
     return selectedRamp.createGradientCanvas(120, theme.controls.height).toDataURL();
   }, [props.selectedRamp, props.reversed]);
 
+  const selectedRampItem = {
+    value: SELECTED_RAMP_ITEM_KEY,
+    label: selectedRampData.name + (props.reversed ? " (reversed)" : ""),
+    image: rampImgSrc,
+    tooltip: selectedRampData.name + (props.reversed ? " (reversed)" : ""),
+  };
+
   const paletteImgSrc = useMemo(() => {
     const visibleColors = props.selectedPalette.slice(0, Math.max(1, props.numCategories));
     const colorRamp = new ColorRamp(visibleColors, ColorRampType.HARD_STOP);
-    return colorRamp.createGradientCanvas(120, theme.controls.height).toDataURL();
+    return colorRamp.createGradientCanvas(120 - 2, theme.controls.height).toDataURL();
```

Comment:
`- 2` for borders? Seeing a fair bit of the number 120 around as well, might be worth making one or both named constants, or else commenting on where `- 2` is coming from here

Reply:
Whoops my magic numbers are escaping